plugins/plugin-chart-echarts/src/BoxPlot/transformProps.ts

```diff
@@ -133,10 +134,9 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
         name: 'boxplot',
         type: 'boxplot',
         avoidLabelOverlap: true,
-        // @ts-ignore
         data: transformedData,
         tooltip: {
-          formatter: param => {
+          formatter: (param: CallbackDataParams) => {
```

Comment:
This looks ok, just curious if there could be a dedicated type for the tooltip formatter params?

Reply:
Not really, formatter does not have a dedicated type